vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@cpuguy83 well, I did some research and didn't found a way of doing it without the certificates...

what I found is that there were too many certificates there, so I removed those which were not needed. https://github.com/bsphere/le_go/pull/10

Any suggestions?